src/material-luxon-adapter/adapter/luxon-date-formats.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {MatDateFormats} from '@angular/material/core';
+
+
+export const MAT_LUXON_DATE_FORMATS: MatDateFormats = {
+  parse: {
+    // Note: this isn't localized like `D`, however Luxon can't parse input against `D`.
+    dateInput: 'L/d/yyyy',
```

Comment:
I don't think this is correct any more and D does work. 

I remember having to fix it in my fork at: https://github.com/andreialecu/ngx-material-luxon/commit/aa7cc93ccbd377264a72420ddb02f2e6a28aa92e